src/pudl/transform/epacems.py

```diff
@@ -207,10 +205,10 @@ def transform(raw_df: pd.DataFrame, pudl_engine: sa.engine.Engine) -> pd.DataFra
 
     """
     return (
-        raw_df.fillna({"gross_load_mw": 0.0, "heat_content_mmbtu": 0.0})
```

Comment:
Just wanted to make sure that this was being tracked as part of https://github.com/catalyst-cooperative/pudl/issues/604

Reply:
Thanks for flagging!

Reply:
I've connected this PR to that issue.